lib/common/constants.dart

```diff
@@ -14,6 +14,8 @@ class ZOColors {
   static const disabledButtonChild = Color.fromRGBO(28, 27, 31, 0.16);
   static const infoSnackBarBackground = Color.fromRGBO(54, 47, 46, 1);
   static const onInfoSnackBarBackground = Color.fromRGBO(255, 251, 238, 1);
+  static const lightSuccessColor = Color.fromRGBO(231, 240, 204, 1);
+  static const successColor = Color.fromRGBO(135, 179, 0, 1);
```

Comment:
Prijde mi "color" tam duplicitni

A pridal bych i "on" barviku:
```suggestion
  static const success = Color.fromRGBO(135, 179, 0, 1);
  static const successLight = Color.fromRGBO(231, 240, 204, 1);
  static const onSuccess = Color.fromRGBO(255, 255, 255, 1);
```